### DISCUSSION TITLE
Agent forwarding: agent refused operation

### DISCUSSION THREAD

**Discussion by naggie** [blinksh/blink#1754] [Support]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

`Blink 16.1.6`
`iOS 16.3.1`

RSA key generated on device, with `ForwardAgent YES`

### Describe the bug

I cannot SSH via SSH Agent forwarding despite providing `-A` to the ssh command or `ForwardAgent YES` in the host configuration as I get `agent refused operation`
![image](https://user-images.githubusercontent.com/208440/231715985-615e33c8-f626-47e4-9608-24ac4b180b43.png)

To make sure it's not just github I tried another host on my LAN that I can ssh to directly using the local key:
![image](https://user-images.githubusercontent.com/208440/231716200-7fff6dd5-df20-485d-88e4-63c1a60f89dd.png)

As described for an older version of blink in another issue, I tried running the agent manually -- and I got a missing symbol error which could help diagnose the bug. I'm aware (via docs) that I shouldn't have to run the agent manually nowadays.

![image](https://user-images.githubusercontent.com/208440/231716540-4c25a71b-0533-4df2-92d0-d27cfcfa5e67.png)

Thanks! This is a really nice project and it will make my iPhone much more useful when travelling when this is working!

**Comment by carloscabanero** [comment]
Hey @naggie ! Blink includes its own ssh agent already embedded inside the connection, and it is actually used for regular authentication. You do not need to start ssh-agent, just make sure you load the key with `ssh-add id_rsa`. That should do the trick.

**Comment by naggie** [comment]
Thanks @carloscabanero I had completely missed the `ssh-add` step which is indeed in the documentation. Sorry about that! Got it working now.
